docker/config/local_dev.env

```diff
@@ -77,8 +77,9 @@ processor.minidumpstackwalk.symbol_tmp_path=/tmp/symbols/tmp
 # a long time
 processor.minidumpstackwalk.kill_timeout=30
 
-# Set symbols_urls to something helpful for local dev
-processor.minidumpstackwalk.symbols_urls=https://symbols.mozilla.org/
+# Set symbols_urls to two symbol servers so we make sure we can specify multiple
+# --symbols-url values in minidump-stackwalk
+processor.minidumpstackwalk.symbols_urls=https://symbols.mozilla.org/try,https://symbols.stage.mozaws.net/try
```

Comment:
This makes it more likely we catch something like this in local dev environment and CI before it hits stage. Plus this adds `try` which expands the kinds of crash reports we can test.